xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-macro/macroEditor.js

```diff
@@ -723,7 +724,10 @@ define(
           load.call(this, input, macroEditor);
         }
       });
-      submitButton.on('click', function(event) {
+      modal.find('.modal-content')[0].addEventListener('submit', function(event) {
```

Comment:
why not using jQuery style here? `.on('submit', callback)`?

Reply:
While I agree moving to vanillajs could be nice, it needs to be discussed on a proposal first.
For now, I'm +1 to keep using jQuery for consitency.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2970/commits/81baeca8ac3a25b91ccd020aa807399c5e8d7a25 :+1: